EasyCosmos/ObjectActivator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace EasyCosmos
+{
+    class ObjectActivator
+    {
+        public static object CreateInstance(Type expectedType)
+        {
+            return Expression.Lambda<Func<object>>(Expression.New(expectedType)).Compile().Invoke();
```

Comment:
Any reason this over the built in Activator.CreateInstance() ?

Reply:
performance gains, should be faster.
https://stackoverflow.com/questions/6582259/fast-creation-of-objects-instead-of-activator-createinstancetype